JavaScript/blog.js

```diff
@@ -0,0 +1,63 @@
+blogEntryStore = [];
+count=1;
+
+function Blog(title,category,author,authorUrl,publishedOn,body){
+  this.title = title;
+  this.category=category;
+  this.author=author;
+  this.authorUrl=authorUrl;
+  this.publishedOn=publishedOn;
+  this.body=body;
+  this.id=count++;
+
+  // this.rawData=[];
+  //   blog.rawData.push(this);
+    blogEntryStore.push(this);
+
+}
+
+
+Blog.prototype.toHTML = function () {
+  $contentSection = $('#contentSection');
+
+  $contentSection.append(
```

Comment:
your .toHTML function uses string-building, which is fine for the beginning! check the pair programming exercise to see how it uses the jQuery .clone method (if you haven't already done so). this will help prepare you for handlebars and other data functions later down the line...